cpp/inc/bond/ext/grpc/bond_utils.h

```diff
@@ -40,10 +40,10 @@ namespace bond { namespace ext { namespace gRPC { namespace detail
 
         for (blob& data : *buffers)
         {
-            data = blob_own(std::move(data));
+            data = blob_prolong(std::move(data));
 
             slices.emplace_back(
-                const_cast<void*>(data.data()),
+                const_cast<void*>(data.data()), // The buffer is not expected to be modified.
```

Comment:
Minor: "modified, but we have to const_cast<void*> because the grpc::Slice ctor only takes void*"
This way we explain why the cast is needed too without requiring the reading to have memoryized the grpc::Slice ctor overloads.